src/tox/session/cmd/run/common.py

```diff
@@ -163,6 +163,19 @@ def __call__(
             help="skip package installation for this run",
             action="store_true",
         )
+        list_deps = parser.add_mutually_exclusive_group()
```

Comment:
I'd like the is_ci to be the default selection and be moved here.

Reply:
That seems a bit counter-intuitive to me, but I've done so :smiley:

Reply:
I just want to avoid the surprise that the user set --no-list-dependencies and then CI=1, or other way around and things don't work as expected 😊